### PR TITLE
fix logout etc

### LINKED ISSUE #20: Fix header layout
Login control gets wrapped.
Also, it should use a `<paper-button>` for login and logout. See the wireframes: https://confluence.pgs-soft.com/display/FRON/Wireframes

### LINKED ISSUE #24: Fix login button handler
[This line](https://github.com/PGS-dev/framapp-polymer/blob/master/src/framapp-polymer-app/fp-login-component.html#L22) should use Polymer-style event handling.

### PR DESCRIPTION
**Pull request by szymczakk**
fix #20 fix #24

### CODE REVIEW

File: src/framapp-polymer-app/framapp-polymer-app.html [hunk 1]
```diff
@@ -73,7 +77,7 @@
                   <div main-title>FramApp</div>
                 </a>
               </div>
-              <fp-login-component></fp-login-component>
+              <fp-login-component class="fr"></fp-login-component>
```
Comment (tpluscode) [inline src/framapp-polymer-app/framapp-polymer-app.html]: you changed the style so class is not needed anymore

File: src/framapp-polymer-app/framapp-polymer-app.html [hunk 2]
```diff
@@ -77,7 +77,7 @@
                   <div main-title>FramApp</div>
                 </a>
               </div>
-              <fp-login-component class="fr"></fp-login-component>
```
Comment (szymczakk) [inline src/framapp-polymer-app/framapp-polymer-app.html]: :+1:

File: .vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (szymczakk) [inline .vscode/tasks.json:1]: this file were not suppose to be here 😢

**Comment by tpluscode** [comment]
We discussed this already. I don't mind ;-)

⁣-- 
Tomasz Pluskiewicz ​

File: .vscode/tasks.json (hunk 3 above)
Comment (tpluscode) [reply .vscode/tasks.json:1]: We discussed this already. I don't mind ;-)

**Review by tpluscode — COMMENTED** [review]
@Tatarak @szymczakk Damn, I couldn't understand why the layout didn't work as expected - the were incomplete imports and `<app-toolbar>`! See how simpler styles are now required. 

There is just that `poiner-events` trick. Without it the toolbar swallows any clicks.

**Pull request merged**